### PR TITLE
Add instant triggers for "new issue" and "updated issue"

### PR DESCRIPTION
**Pull request by leelasn**
The final two — adds instant triggers for "new issue" and "updated issue"

Closes USP-6765, USP-6767

### CODE REVIEW

File: src/triggers/issueV2.ts [hunk 1]
```diff
@@ -0,0 +1,329 @@
...
+      label: "Label",
+      key: "labelId",
+      helpText: "Label which was assigned to the issue.",
+      dynamic: "label.id.name",
+      altersDynamicFields: true,
+    },
+    {
+      required: false,
+      label: "Project",
+      // This key is snake-cased for backwards compatibility with the dynamic field selector for project milestone
```
Comment (tommoor) [inline src/triggers/issueV2.ts:283]: Not sure I follow, backwards compatible how – this is brand new trigger?

Comment (leelasn) [reply src/triggers/issueV2.ts:283]: Look at the `projectMilestone` trigger under `src/triggers` that we use to populate the dynamic selector

Comment (tommoor) [reply src/triggers/issueV2.ts:283]: Could update that to support either/or I guess rather than compromising this new one 🤔

Comment (leelasn) [reply src/triggers/issueV2.ts:283]: That one is used in the polling trigger version and renaming a bundle variable is a breaking change. I could just create v2 instead to use here?

Comment (leelasn) [reply src/triggers/issueV2.ts:283]: Oh, I see what you mean: check for either variable there

**Review by tommoor — APPROVED** [review]

**Pull request merged**